vignettes/installation.Rmd

```diff
@@ -83,19 +83,19 @@ Graphical user interface for R interpreter will start in a new window:
 include_graphics("fig/RGui1.png")
 ```
 
-## 2.2. Installing *seroincidence* package
+## 2.2. Installing *serocalculator* package
 
 Since this is a new installation of R, the **serocalculator** package must be installed before first 
 use. The package can be installed either through a menu option in the graphical user interface (GUI) 
 or with a command executed in the R console. Both options are presented:
 
 ### 2.2.1. Package installation from the graphical user interface
```

Comment:
This subsection is out of date; remove?

Reply:
Replaced with updated installation instructions